BUILD.md

```diff
@@ -52,7 +52,7 @@ Follow the instructions below to execute a Docker-based build and execution.
 3. On a successful run, you should see the message `BUILD SUCCESS`
    followed by a repetitive message `Enclave manager sleeping for 10 secs`
 4. Open a Docker container shell using following command
-   `sudo docker exec -it tcf bash`
+   `sudo docker exec -it avalon-shell bash`
```

Comment:
Please make the documenttaion changes in below files as well.
docs/TestingFabricProxyModel.md
docs/BesuProxyModel.md
docs/workload-tutorial/README.md

Reply:
Done